src/08-Security.mdx

```diff
@@ -0,0 +1,95 @@
+---
+name: 08. Security
+---
+
+import Authors from './helpers/Authors';
+
+# Security
+Level: *Hidden Hippopotamus*
+
+The security aspects of your project are just as important as performance and UX.
+Although more security issues refer to the Backend it's important for any Frontend Developer to know at least the theory behind all those security issues.
```

Comment:
to weak for me. it indicates there are none issues in frontend and this chapter is just to kno the theory of backend security issues.

Reply:
Agreed. That sentence was pointing to the OWASP Top10 list. I defused the sentence and moved it below the owasp top10 link.

Reply:
"knows the few that are related to frontend very well, and at least the theory for those which are Backend related"?